tests/metrics/verifyMysqlLogLevel_test.js

```diff
@@ -1,22 +1,26 @@
 const assert = require('assert');
-const faker = require('faker');
-
-const { adminPage } = inject();
 
 Feature('Integration tests for Mysql Exporter PMM Agent and Log Level');
-const pmmFrameworkLoader = `bash ${adminPage.pathToFramework}`;
 
 const connection = {
   // eslint-disable-next-line no-inline-comments
-  port: '3307', // This is the port used by --setup-pmm-ps-integration --pmm2 --query-source=slowlog --ps-version=8.0
-  container_name: 'ps_pmm_8.0',
+  port: '3307', // This is the port used by --database ps,QUERY_SOURCE=slowlog
```

Comment:
Kindly update this to 3317
It was changed based on @peterSirotnak request on https://github.com/Percona-Lab/qa-integration/pull/52/files